staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2579,6 +2615,20 @@ type PodCondition struct {
 	Message string `json:"message,omitempty" protobuf:"bytes,6,opt,name=message"`
 }
 
+// ResourcesResizeStatus shows status of resources resize of a pod's containers.
+type ResourcesResizeStatus string
```

Comment:
I'd call this `PodResizeStatus` - it's not per-resource

Reply:
Yes that is clearer, I'll fix it.

Reply:
fixed in commit https://github.com/kubernetes/kubernetes/pull/102884/commits/b60c5d045bb85eacae064e209994c4afe22a4e5f